NEWS.md

```diff
@@ -4,6 +4,8 @@
 
 * The package will now log a backtrace for errors and warnings that occur during tuning. When a tuning process encounters issues, see the new `trace` column in the `collect_notes(.Last.tune.result)` output to find precisely where the error occurred (#873).
 
+* When automatic grids are used, [dials::grid_space_filling()] is now used (instead of [dials::grid_latin_hypercube()]).
```

Comment:
Could we add a sentence or two on why we're doing this and also note that we still use a latin hypercube to generate grids for the GP?